library/classes/Document.class.php

```diff
@@ -283,6 +283,16 @@ public function has_expired()
         return false;
     }
 
+    public function can_patient_access($pid)
```

Comment:
will be good to get this PR asap since I just realized i just brought code into rel-700 that uses this function :)